estate/models/estate_property.py

```diff
@@ -20,39 +22,70 @@ class EstateProperty(models.Model):
         ('east', 'East'),
         ('west', 'West')
     ])
+    status = fields.Selection([
+        ('new', 'New'),
+        ('sold', 'Sold'),
+        ('canceled', 'Canceled')], default='new', required=True)
```

Comment:
```suggestion
        ('new', "New"),
        ('sold', "Sold"),
        ('canceled', "Canceled")], default='new', required=True)
```
User facing strings in double quotes